stytch/consumer/api/sessions.py

```diff
@@ -0,0 +1,355 @@
+
+    # MANUAL(authenticate_jwt)(SERVICE_METHOD)
+    # ADDIMPORT: from typing import Any, Dict, Optional
+    # ADDIMPORT: import jwt
+    # ADDIMPORT: import time
+    def authenticate_jwt(
+        self,
+        session_jwt: str,
+        max_token_age_seconds: Optional[int] = None,
+        session_custom_claims: Optional[Dict[str, Any]] = None,
+    ) -> Optional[Session]:
+        """Parse a JWT and verify the signature, preferring local verification
+        over remote.
+
+        If max_token_age_seconds is set, remote verification will be forced if the
+        JWT was issued at (based on the "iat" claim) more than that many seconds ago.
+
+        To force remote validation for all tokens, set max_token_age_seconds to
+        zero or use the authenticate method instead.
+        """
+        # Return the local_result if available, otherwise call the Stytch API
+        return (
+            self.authenticate_jwt_local(
+                session_jwt=session_jwt,
+                max_token_age_seconds=max_token_age_seconds,
+            )
+            or self.authenticate(
+                session_custom_claims=session_custom_claims, session_jwt=session_jwt
+            ).session
+        )
+
+    async def authenticate_jwt_async(
+        self,
+        session_jwt: str,
+        max_token_age_seconds: Optional[int] = None,
+        session_custom_claims: Optional[Dict[str, Any]] = None,
+    ) -> Optional[Session]:
+        """Parse a JWT and verify the signature, preferring local verification
+        over remote.
+
+        If max_token_age_seconds is set, remote verification will be forced if the
+        JWT was issued at (based on the "iat" claim) more than that many seconds ago.
+
+        To force remote validation for all tokens, set max_token_age_seconds to
+        zero or use the authenticate method instead.
+        """
+        # Return the local_result if available, otherwise call the Stytch API
+        return (
+            self.authenticate_jwt_local(
+                session_jwt=session_jwt,
+                max_token_age_seconds=max_token_age_seconds,
+            )
+            or (
+                await self.authenticate_async(
+                    session_custom_claims=session_custom_claims, session_jwt=session_jwt
+                )
+            ).session
+        )
+
+    # ENDMANUAL(authenticate_jwt)
+
+    # MANUAL(authenticate_jwt_local)(SERVICE_METHOD)
+    # ADDIMPORT: import jwt
+    # ADDIMPORT: import time
+    # ADDIMPORT: from stytch.consumer.models.sessions import Session
+    def get_jwks_client(self) -> jwt.PyJWKClient:
+        data = {"project_id": self.sync_client.project_id}
+        jwks_url = self.api_base.url_for("v1/sessions/jwks/{project_id}", data)
+        return jwt.PyJWKClient(jwks_url)
+
+    def authenticate_jwt_local(
+        self,
+        session_jwt: str,
+        max_token_age_seconds: Optional[int] = None,
+        leeway: int = 0,
+    ) -> Optional[Session]:
```

Comment:
I see we used to return an `AuthenticateResponse` for this function - why'd we change to just `Session`?